sentinel.conf

```diff
@@ -145,8 +142,8 @@ sentinel down-after-milliseconds mymaster 30000
 #   user worker +@admin +@connection ~* on >ffa9203c493aa99
 #
 # For more information about ACL configuration please refer to the Redis
-# website at https://redis.io/topics/acl and redis server configuration 
-# template redis.conf.
+# website at https://redis.io/topics/acl and valkey server configuration 
```

Comment:
As long as our website will have the same URLs, I think this link will work. This pages are defined as files in valkey-doc.

```suggestion
# website at https://valkey.io/topics/acl and valkey server configuration 
```

Reply:
Address your comments, and I update redis.io to valkey.io in 2 places, pls take a look. Thanks